docs/wiki/radios/sem-70.md

```diff
@@ -0,0 +1,83 @@
+---
+title: SEM 70
+---
+
+The SEM 70 (Sender/Empfänger, mobil SEM 70) is a manpack radio for military purposes in the German Army. It was issued in 1984. As a modern radio it can transmit voice and data. The frequency band is from 30 to 79,975 MHz. The channel spacing is 25 KHz by default. It features a manual channel (_Handwahl (HW)_) and an automatic channel (_Automatische Kanalwahl (AKW)_) mode. 
+
+## Technical Data
+
+- **Frequency band:** 30,000 to 79,975 MHz
+- **Channel spacing:** 25 KHz (default) or 50 kHz
+- **Channel presets:** Manual channel and up to 10 memory slots for automatic channel selection
+- **Operating mode:** Simplex or Semiduplex
+- **Operating voltage:** 16.8 V
+- **Transmitting capacity:** 4 W or 0.4 W
+- **Antenna:** 0,4 m or 1,03 m
+
+## Primer On Automatic Channel Mode (AKW)
+
+While in automatic channel mode, the operator can select from 10 Channels. Each channels combines 16 different frequencies, which can't be changed by the operator. While no transmission is active, every radio continously scans all 16 frequencies in a loop. If a transmission is initiated, the sending radio randomly selects a frequency on which all receiving radios lock on. IRL this usually takes about 0.7 Seconds, in ACRE2 there is no such delay (may be added at a later point).
+However, to establish a valid connection between two or more radios, the following parameters must be the same on all involved radios
+- Channel Setup (Channels/Frequencies must be identical for each radio)
+- Memory Slot (Channel)
+- Network Identifier (Three digit number)
+
+## Basic Operation:
+
+### Interface
+
+{% include image.html file="radios/sem-70_interface.jpg" alt="SEM 70 Interface" %}
+
+### Volume Knob
+
+Sets the overall volume of the radio, 6 settings are available.
+
+### Power Knob
+
+The radio features two different power settings. On high power setting (_GR_) the output is 4 Watts and on low power setting (_KL_) the output is 0.4 Watts. If the knob is set to off (_AUS_), the radio can't transmit or receive anything.
```

Comment:
I am thinking code ticks might be better for button abbreviations. SEM 52 SL doesn't use either, so either of them are an improvement. :)